sns_ik_lib/include/sns_ik/fsns_velocity_ik.hpp

```diff
@@ -36,22 +36,22 @@ namespace sns_ik {
 
 class FSNSVelocityIK : public SNSVelocityIK {
   public:
-    FSNSVelocityIK(int dof, Scalar loop_period);
+    FSNSVelocityIK(int dof, double loop_period);
     virtual ~FSNSVelocityIK() {};
 
     // Optimal SNS Velocity IK
-    virtual Scalar getJointVelocity(VectorD *jointVelocity, const std::vector<Task> &sot,
-                  const VectorD &jointConfiguration);
+    virtual double getJointVelocity(Eigen::VectorXd *jointVelocity, const std::vector<Task> &sot,
```

Comment:
Same as above, with explicit declarations of namespace, we should remove the `using` declaration.